curriculum/class-00/lab-a/starter-code/js/app.js

```diff
@@ -71,9 +71,9 @@ function handleClick(event) {
 }
 
 function showList() {
-  for(var i = 0; i < allProducts.length; i++) {
-    var liEl = document.createElement('li');
-    liEl.textContent = allProducts[i].name + ' has ' + allProducts[i].votes + ' votes in ' + allProducts[i].views + ' views';
+  for(const i = 0; i < allProducts.length; i++) {
```

Comment:
you want to use a `let` rather than a `const` in a for loop, otherwise the loop cant run